portal-api/src/v1/routes.js

```diff
@@ -33,6 +33,9 @@ const users = require('./users/routes');
 const gef = require('./gef/routes');
 
 const openRouter = express.Router();
+const authRouter = express.Router();
```

Comment:
Can we move the create router and the `use` statements to be next to each other? For someone new to this file currently it would be easy for them to miss that `authRouter` has an xss check because it's so much further down the file.